spec/latest/core/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 <title>Hydra Core Vocabulary</title>
 <meta charset="utf-8">
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch|gated"></script>
```

Comment:
ReSpec requires Promises, which makes it incompatible with IE anyway. And I wouldn't fight over fetch.